spec/features/admin_categories_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+describe 'admin categories', type: :feature do
+
+  before(:each) do
+    User.create(full_name: "vanessa", display_name: "admin", email: "admin@email.com", role: 1, password: "dinnerdash")
+    visit login_path
+    click_on "Sign Up/Login"
+    page.fill_in('Email', with: 'admin@email.com')
+    page.fill_in('Password', with: 'dinnerdash')
+    click_on "Log in"
+    visit admin_categories_path
+  end
+
+  it 'creates new categories' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.